packages/graphql-codegen-cli/tests/generate-and-save.spec.ts

```diff
@@ -281,37 +276,34 @@ describe('generate-and-save', () => {
           false
         );
       } catch {
-        expect(consoleErrorMock.mock.calls[0][0]).toBeSimilarStringTo(
-          '[FAILED] Failed to load schema from ./tests/test-files/schema-dir/error-schema.graphql:'
-        );
-        expect(consoleErrorMock.mock.calls[0][0]).toBeSimilarStringTo(
-          '[FAILED] Syntax Error: Expected Name, found "!".'
-        );
-        // We can only use partial file path to the error file, because the error contains absolute path on the host machine
-        expect(consoleErrorMock.mock.calls[0][0]).toBeSimilarStringTo(
-          '/tests/test-files/schema-dir/error-schema.graphql:2:15'
-        );
-        expect(consoleErrorMock.mock.calls[0][0]).toBeSimilarStringTo(`
-          [FAILED] 1 | type Query {
-          [FAILED] 2 |   foo: String!!
-          [FAILED]   |               ^
-          [FAILED] 3 | }
-          [FAILED]
-          [FAILED] GraphQL Code Generator supports:
-          [FAILED]
-          [FAILED] - ES Modules and CommonJS exports (export as default or named export "schema")
-          [FAILED] - Introspection JSON File
-          [FAILED] - URL of GraphQL endpoint
-          [FAILED] - Multiple files with type definitions (glob expression)
-          [FAILED] - String in config file
-          [FAILED]
-          [FAILED] Try to use one of above options and run codegen again.
+        const cwd = process.cwd(); // cwd is different for every machine, remember to replace local path with this after updating snapshot
+        expect(outputErrorMock.mock.calls[0][0]).toMatchInlineSnapshot(`
+          "[31m[FAILED][39m Failed to load schema from ./tests/test-files/schema-dir/error-schema.graphql:
+          [31m[FAILED][39m Syntax Error: Expected Name, found "!".
+
+          [31m[FAILED][39m ${cwd}/tests/test-files/schema-dir/error-schema.graphql:2:15
```

Comment:
`cwd` is overwritten to the machine's actual path if we update the snapshot tests of this file 
i.e. the person updating this test would have to replace this manually. This could be annoying.

However, this is rarely updated so let's try this for now.